action/build/get.go

```diff
@@ -19,9 +19,14 @@ func (c *Config) Get(client *vela.Client) error {
 	// set the pagination options for list of builds
 	//
 	// https://pkg.go.dev/github.com/go-vela/sdk-go/vela?tab=doc#ListOptions
```

Comment:
Could we update the link here for the new type being used?

```suggestion
	// https://pkg.go.dev/github.com/go-vela/sdk-go/vela?tab=doc#BuildListOptions
```